src/super_gradients/training/datasets/segmentation_datasets/coco_segmentation.py

```diff
@@ -32,15 +32,20 @@ def __init__(self, dataset_classes_inclusion_tuples_list: list = None, *args, **
 
         # OVERRIDE DEFAULT AUGMENTATIONS, IMG_SIZE, CROP SIZE
         dataset_hyper_params = kwargs['dataset_hyper_params']
-        kwargs['img_size'] = dataset_hyper_params['img_size'] if 'img_size' in dataset_hyper_params.keys() else 608
-        kwargs['crop_size'] = dataset_hyper_params['crop_size'] if 'crop_size' in dataset_hyper_params.keys() else 512
+        kwargs['img_size'] = get_param(dataset_hyper_params, 'img_size', 608)
+        kwargs['crop_size'] = get_param(dataset_hyper_params, 'crop_size', 512)
         # FIXME - Rescale before RandomRescale is kept for legacy support, consider removing it like most implementation
         #  papers regimes.
-        kwargs["image_mask_transforms_aug"] = transform.Compose([RandomFlip(),
-                                                                 Rescale(long_size=kwargs["img_size"]),
-                                                                 RandomRescale(scales=(0.5, 2.0)),
-                                                                 PadShortToCropSize(crop_size=kwargs['crop_size']),
-                                                                 CropImageAndMask(crop_size=kwargs['crop_size'], mode="random")])
+        default_image_mask_transforms_aug = transform.Compose([RandomFlip(),
+                                                               Rescale(long_size=kwargs["img_size"]),
+                                                               RandomRescale(scales=(0.5, 2.0)),
+                                                               PadShortToCropSize(crop_size=kwargs['crop_size']),
+                                                               CropImageAndMask(crop_size=kwargs['crop_size'], mode="random")])
+        kwargs["image_mask_transforms_aug"] = get_param(dataset_hyper_params, "image_mask_transforms_aug",
+                                                        default_image_mask_transforms_aug)
+        image_mask_transforms = get_param(dataset_hyper_params, 'image_mask_transforms', False)
```

Comment:
I know this seems like a small thing...but lets change this to image_mask_transforms = get_param(dataset_hyper_params, 'image_mask_transforms') which will set it by default to None if the key doesn't exist. Just so when somebody debugs this they won't get confused that image_mask_transforms is some flag for something else.

Reply:
Agree, added in 20280ca